### ISSUE TITLE
Question: Is it possible to download the training data as well as the pre-trained model? 

### ISSUE THREAD

**Issue by hradec** [mireiffe/individual_tooth_segmentation#1]
Thanks for this great work!! It's exactly what I was looking for a long time! 

I'm also interested in use this project to segment teeth in open mouth photos as well, so I guess I need to complement the original dataset with open mouth photos and labels.

Some information on how to train would be amazing too, if possible! 

Anyhow, thanks for sharing this work... just amazing!
